coalib/output/ConsoleInteraction.py

```diff
@@ -560,26 +562,35 @@ def choose_action(console_printer, actions):
     :return:                Return choice of action of user.
     """
     while True:
-        console_printer.print(format_lines('*0: ' +
-                                           'Do nothing'))
+        dict_actions = {'(O)pen file': 'O', '(A)pply patch': 'A',
+                        'Print (M)ore info': 'M', 'Add (I)gnore comment':
+                            'I', 'Print (D)ebug message': 'D',
+                            'Print Aspec(T) Information': 'T'}
```

Comment:
nonononono we're not hardcoding this are we? :) Can't we automatically determine the right character from the documentation that we already have? Add the `(` and `)` around it automatically such that it is unique and when writing an action I don't have to be careful not to choose a wrong character?